google-cloud-storage/src/main/scala/akka/stream/alpakka/googlecloud/storage/StorageObject.scala

```diff
@@ -229,13 +243,14 @@ final class StorageObject private (
     s"retentionExpirationTime = $retentionExpirationTime," +
     s"timeStorageClassUpdated = $timeStorageClassUpdated," +
     s"cacheControl = $cacheControl," +
-    metadata.fold("")(m => s"metadata = $m,") +
-    componentCount.fold("")(cc => s"componentCount = $cc,") +
-    kmsKeyName.fold("")(kkn => s"kmsKeyName = $kkn,") +
-    customerEncryption.fold("")(ce => s"customerEncryption = $ce,") +
-    owner.fold("")(o => s"owner = $o,") +
-    acl.fold("")(acls => acls.mkString("[", ",", "]")) +
-    ")"
+    s"customTime = $customTime"
+  metadata.fold("")(m => s"metadata = $m,") +
+  componentCount.fold("")(cc => s"componentCount = $cc,") +
+  kmsKeyName.fold("")(kkn => s"kmsKeyName = $kkn,") +
+  customerEncryption.fold("")(ce => s"customerEncryption = $ce,") +
+  owner.fold("")(o => s"owner = $o,") +
+  acl.fold("")(acls => acls.mkString("[", ",", "],")) +
+  ")"
```

Comment:
Not sure why this change was needed, but you should add any missing fields to the `toString`.